handler/src/main/java/io/netty/handler/ssl/OpenSslContext.java

```diff
@@ -507,73 +497,74 @@ static long toBIO(PrivateKey key) throws Exception {
         if (key == null) {
             return 0;
         }
-        ByteBuf buffer = Unpooled.directBuffer();
+
+        ByteBufAllocator allocator = ByteBufAllocator.DEFAULT;
```

Comment:
The alloc is passed ~2 lines further down into the `toBIO()` call. Feels more appropriate to have it as a temp variable and ideally passed in as method argument (but that would break existing API).